networking.md

```diff
@@ -0,0 +1,88 @@
+# Filecoin Network
+
+The filecoin network is built using libp2p building blocks, as transports and protocols, as well as some additional filecoin specific protocols as outlined in [Network Protocols](./network-protocols.md).
+
+## Required Protocols
+
+Every full node must support the following libp2p protocols:
+
+ - [gossipsub](https://github.com/libp2p/specs/tree/master/pubsub/gossipsub) ([for data announcements](./data-propagation.md))
+ - [bitswap] ([for data exchange](./data-propagation.md))
+ - Filecoin specific Protocols:
+   - Hello Handshake
+   - StorageDeal
+   - BlockSync
```

Comment:
What about the other protocols defined?

Reply:
I'd link these into the relevant network-protocols.md portions

Reply:
as said, we have a linking issue, waiting for that to be resolved.